src/app/CMakeLists.txt

```diff
@@ -139,11 +139,7 @@ endif()
 if (GUI)
     target_link_libraries(qbt_app PRIVATE qbt_gui)
     if ((CMAKE_SYSTEM_NAME STREQUAL "Windows") OR (CMAKE_SYSTEM_NAME STREQUAL "Darwin"))
-        if (Qt5_VERSION VERSION_LESS 5.14)
-            set_property(TARGET qbt_app APPEND PROPERTY QT_PLUGINS Qt5::QSvgIconPlugin Qt5::QSvgPlugin)
-        else()
-            qt_import_plugins(qbt_app INCLUDE Qt5::QSvgIconPlugin Qt5::QSvgPlugin)
-        endif()
+        qt_import_plugins(qbt_app INCLUDE Qt6::QSvgIconPlugin Qt6::QSvgPlugin)
```

Comment:
Does someone know why this line causes the following warnings?
>Warning: plug-in Qt6::QSvgIconPlugin is not known to the current Qt installation.
Warning: plug-in Qt6::QSvgPlugin is not known to the current Qt installation.

Reply:
Maybe this helps:
https://bugreports.qt.io/browse/QTBUG-89643

Reply:
I noticed this bug report. Unfortunately, I don't see any useful information there. In addition, it belongs to a statically linked Qt.

Reply:
I noticed this bug report. Unfortunately, I don't see any useful information there. In addition, it belongs to a statically linked Qt.

Reply:
Just found in Qt docs:
>Note: This command imports plugins from static Qt builds only.
On shared builds, it does nothing

Reply:
@glassez 

I had fixed those warnings in https://github.com/glassez/qBittorrent/pull/1 ...

Reply:
Doesn't seem that adding `NOT MSVC_RUNTIME_DYNAMIC` condition is correct way to fix it.

Reply:
@glassez

Why? OK, maybe it fails in the corner case where you're building with static libraries (including Qt) but using `-MD`, is that it (such as the case with `vcpkg`'s unofficial `x64-windows-static-md` triplet)? If that's the case, to fix it fully we need a way to ask Qt itself if it was built statically or not (I think this should be easy, I'll look into it).

My fix was based on the assumption that if you're building with `MSVC_RUNTIME_DYNAMIC=OFF` you're also building with static libraries, which I think is fair, but yeah, it leaves out that corner case.

Reply:
Could https://cmake.org/cmake/help/latest/prop_tgt/TYPE.HTML help?

Reply:
@glassez Thanks, the `TYPE` solution is confirmed to work. It is now implemented in https://github.com/glassez/qBittorrent/pull/1.